pages/glossary/keyboard-actionable-element.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Keyboard Actionable Element
+key: keyboard-actionable-element
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An HTML element is _keyboard actionable_ if all the following are true:
+
+- the element is part of [sequential focus navigation][]; and
+- the element is [visible][] when [focused][]; and
+- the element can be [activated][] by use of keyboard.
```

Comment:
I am not sure I understand the objection.
Is it possible to use only your keyboard and trigger the activation of that element?

Reply:
Don't think this should've been resolved. How does someone determine if an element can be activated by use of a keyboard? I don't think the definition of "activated" says anything about that. What even is an "activated" element? Is it an event with a click event listener? Is it any element that can be the target of a click event, which is essentially every element?

Reply:
Isn't it enough for the tester to try to evaluate the condition by attempting to activate the element with the keyboard and compare the outcome of the keyboard press with the expected behaviour? 
In a manual testing methodology I suppose that is what you would do to check if the element is in the sequential focus navigation order (press tab on the keyboard and check if the element gets focused) and to check if the element is visible when focused (press tab and ...)

Reply:
The definition of activation (that "activated" was linking to) is:
> Certain elements in HTML have an activation behavior, which means that the user can activate them. This is always caused by a click event.
>
> The user agent should allow the user to manually trigger elements that have an activation behavior, for instance using keyboard or voice input, or through mouse clicks. When the user triggers an element with a defined activation behavior in a manner other than clicking it, the default action of the interaction event must be to fire a click event at the element.

For me, it was obvious that activating an element means triggering said activation behaviour, and that this is the same as firing the click event at the element. 

I've replaced it with "firing a click event", but I think the linked bit of the HTML spec is more technical and confusing than the paragraphs at "activation".


---
Given that this definition is only used in `first-focusable-element-is-link-to-non-repeated-content` which is standalone, if this needs more discussion, I'll try and extract it into a separate PR to not block the rest (I'd like to avoid doing all the extra work of desentangling these, but if it's needed 🤷 ).